### DISCUSSION TITLE
Help with `stabilityDays` and Github CI

### DISCUSSION THREAD

**Discussion by mateusfg7** [renovatebot/renovate#14944] [Request Help] [answered]
I have a renovate bot configured in my github repositorie, with `stabilityDays` for all package updates, but the `renovate/stability-days` CI are running on all my PRs, and never done.

_random PR that was not made by renovate for package update_
![image](https://user-images.githubusercontent.com/40613276/161569479-02563196-6c58-445b-9b20-9fb4188b247b.png)

Is there a config to only run the `renovate/stability-days` in PRs made by Renovate?
Or a config on the github repo to disable some CI in some PRs?

my _**renovate.json**_:
```json
{
  "extends": ["config:base"],
  "commitBodyTable": true,
  "branchPrefix": "deps/",
  "labels": ["dependencies"],
  "stabilityDays": 3,
  "prCreation": "not-pending",
  "semanticCommits": true,
  "packageRules": [
    {
      "matchUpdateTypes": ["minor", "patch", "pin"],
      "automerge": true
    },
    {
      "matchUpdateTypes": ["major"],
      "addLabels": ["d:major"]
    },
    {
      "matchUpdateTypes": ["minor"],
      "addLabels": ["d:minor"]
    },
    {
      "matchUpdateTypes": ["patch"],
      "addLabels": ["d:patch"]
    },
    {
      "matchUpdateTypes": ["pin"],
      "addLabels": ["d:pin"]
    },
    {
      "matchUpdateTypes": ["rollback"],
      "addLabels": ["d:rollback"]
    }
  ]
}
```

**Comment by rarkins** [comment, marked as answer]
They're not running. You or someone with admin access set it as a required status check as part of branch protection rules
